Dockerfile.jinja

```diff
@@ -0,0 +1,41 @@
+FROM python:2.7
```

Comment:
Why is this now a `.jinja` file? That seems like a big change for some additional patches